Dockerfile

```diff
@@ -0,0 +1,137 @@
+# This Dockerfile builds OpenMS, the TOPP tools, pyOpenMS and thidparty tools.
+# It also adds a basic streamlit server that serves a pyOpenMS-based app.
+# hints:
+# build image and give it a name (here: streamlitappsimple) with: docker build --no-cache -t streamlitappsimple:latest . 2>&1 | tee build.log 
```

Comment:
Can this be "streamlitapp" and "streamlitappsimple" at Dockerfile_simple?

Reply:
ah good catch... copy and paste error